plugins/pencilblue/controllers/actions/salesforce/change_email.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
```

Comment:
In the long long run, we should find a better way to bucket all of this - like login/salesforce/... or something.  Not something to do right now though.